docs/getting-started/unreal-editor/index.md

```diff
@@ -72,11 +74,11 @@ See the documentation on [Packaging](../packaging) for more info.
 :::
 
 ## Load Order
-Unreal Engine loads files in PAKs in alphabetical order, so if you want your mod to load after another mod,
+Unreal Engine loads PAK-Files in alphabetical order, so if you want your mod to load after another mod,
 you must make sure your mod's name comes after the other mod's name alphabetically.
 It is best to prefix your mod's name with the letter `z`, ensuring it will load last and overwrite any file you need it to.
 
 ## Viewing PAYDAY 3's PAK files
-In order to view PAYDAY 3's PAK files, you must first obtain the AES key.
-While we cannot distribute the AES key ourselves, you can dump it very easily.
+In order to view PAYDAY 3's PAK files using FModel, you must first obtain the AES key.
```

Comment:
No need to specify FModel here. There's plenty of other PAK-reading utilities that need the AES key, and FModel is already provided as an example below.

Reply:
👍